RoddyCore/src/de/dkfz/roddy/execution/io/SSHExecutionService.groovy

```diff
@@ -157,6 +158,9 @@ class SSHExecutionService extends RemoteExecutionService {
                 c.startSession()
                 t2 = System.nanoTime()
                 logger.sometimes(RoddyIOHelperMethods.printTimingInfo("start ssh client session", t1, t2))
+            } catch (UnknownHostException ex) {
+                logger.severe("Could not setup SSH access with your configuration: The specified host is not available - ${ex.message}")
+                Roddy.exit(255)
```

Comment:
Put exit codes into constants with descriptive names.
Document exit code in RtD.

Reply:
ExitReasons enum with messages and numbers.